content/manuals/ai/cagent/integrations/a2a.md

```diff
@@ -0,0 +1,205 @@
+---
+title: A2A mode
+linkTitle: A2A
+description: Expose cagent agents via the Agent-to-Agent protocol
+keywords: [cagent, a2a, agent-to-agent, multi-agent, protocol]
+weight: 40
+---
+
+A2A mode runs your cagent agent as an HTTP server that other systems can call
+using the Agent-to-Agent protocol. This lets you expose your agent as a service
+that other agents or applications can discover and invoke over the network.
+
+Use A2A when you want to make your agent callable by other systems over HTTP.
+For editor integration, see [ACP integration](./acp.md). For using agents as
+tools in MCP clients, see [MCP integration](./mcp.md).
+
+## Prerequisites
+
+Before starting an A2A server, you need:
+
+- **cagent installed** - See the [installation guide](../_index.md#installation)
```

Comment:
```suggestion
- cagent installed - See the [installation guide](../_index.md#installation)
```
nit: remove bolding (and for the items below) since it's not a ui/tui element you're referring to